src/test/java/emissary/core/IBaseDataObjectHelperTest.java

```diff
@@ -359,6 +359,10 @@ void testAddParentInformationToChildren() {
 
     @Test
     void testFindPreferredDataByRegex() {
+        final List<String> emptyPreferredViews = new ArrayList<>();
+
+        assertThrows(IllegalArgumentException.class, () -> IBaseDataObjectHelper.findPreferredData(null, emptyPreferredViews));
```

Comment:
Wrong method called (probably a copy/paste error)

```suggestion
        assertThrows(IllegalArgumentException.class, () -> IBaseDataObjectHelper.findPreferredData(null,  new ArrayList<>()));
```